docs/privacyidea.md

```diff
@@ -87,11 +87,18 @@ You need to add the authentication source 'privacyidea' to
     'passFieldHint' => 'Password',
     
     /**
-     * Set this to 'true' if you want to use single sign on.
+     * Set SSO to 'true' if you want to use single sign on.
      * All information required for SSO will be saved in the session.
      * After logging out, the SSO data will be removed from the session.
      */
     'SSO' => 'false',
+    
+    /**
+     * Set preferredTokenType to your favourite token type.
+     * If the choosed token will be triggered,
```

Comment:
token **is** triggered, also i think no new is required

Reply:
I don't understand what exactly you mean. Can you change it by yourself?